### PR TITLE
Strike Team Revamp part 2: Role tracking

### PR DESCRIPTION
**Pull request by DeityLink**
- ERT and Deathsquad members now appear in the Round End Information window, as well as on the Admins antagonists panel.
- ERT and Deathsquad roles also now appear in the traitor panel.

### CODE REVIEW

**Comment by CptWad** [comment]
By role do you mean as an "Antag" in the sense of the roundend screen or is there tracking on a forced duty like X is security Y is medical Z is command with A and B as the engineers?

**Comment by DeityLink** [comment]
@CptWad see pics
role
They are not technically "antagonists" (in most cases) but that way everyone can see who was playing them, and what were their objectives (once I add the possibility for admins to easily give objectives to an entire strike team)

**Comment by CptWad** [comment]
right

File: code/modules/admin/verbs/striketeam.dm [hunk 1]
```diff
@@ -119,6 +119,8 @@ var/global/sent_strike_team = 0
 	new_commando.mind.special_role = "Death Commando"
 	ticker.mode.traitors |= new_commando.mind//Adds them to current traitor list. Which is really the extra antagonist list.
 	new_commando.equip_death_commando(leader_selected)
+
+	ticker.mode.deathsquad = new_commando.mind
```
Comment (ComicIronic) [inline code/modules/admin/verbs/striketeam.dm]: +=

**Comment by kilkun** [comment]
Would syndicate black ops get counted under the same endround report antag category as deathsquad? Seems a little silly for them to show up under the NT logo.

**Comment by DeityLink** [comment]
@kilkun Nope, Syndie Elite squads are mostly unconcerned by this update.

File: code/modules/admin/verbs/striketeam.dm (hunk 1 above)
Comment (DeityLink) [inline code/modules/admin/verbs/striketeam.dm]: oh mah god

Comment (Iamgoofball) [inline code/modules/admin/verbs/striketeam.dm]: hivemind deathsquads

feature

**Pull request merged**